b42_prepush_check.sh

```diff
@@ -1,22 +1,27 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-RED='\033[0;91m'
-GREEN='\033[0;32m'
-NC='\033[0m' # No Color
+# Needed for type builtin and globstar shell option
+[ -z "$BASH_VERSION" ] && { echo "Script must be run with bash!"; exit 1; }
 
-git_root=$(git rev-parse --show-toplevel)
+RED='\033[91m'
+GREEN='\033[32m'
+NC='\033[m' # No Color
+
+git_root="$(git rev-parse --show-toplevel)"
 
 summary_status=""
 
-pushd "$git_root"
+pushd -- "$git_root" >/dev/null || {  echo "Couldn't change directory!"; exit 2; }
 echo "== NORM"
-norminette
+"$(type -P norminette)"
```

Comment:
sorry, I didn't get that part.
We are enforcing to use `norminette` as the binary for the case when user has alias with the same name, right?
Is it often a case that user have such an alias? if so, is it a case when `norminette` alias works differently?

Reply:
I've seen this pattern in some repos, that's why I added it, but I did some further research and I think it should be fine _not_ doing it. I think functions and aliases won't be a problem unless someone has special shell options set.
I'm gonna change it back for readability, but it's a good thing to know when you want to force a path lookup

Reply:
```suggestion
norminette
```